apps/models.go

```diff
@@ -346,7 +346,7 @@ type PurposeTemplate struct {
 	LegalBasisRestriction string `yaml:"legalBasisRestriction,omitempty" json:"legalBasisRestriction,omitempty"`
 }
 
-type Tcf struct {
+type Vendor struct {
```

Comment:
what about 

legIntPurposes - string array
flexiblePurposes - string array
usesCookies - bool
cookieMaxAgeSeconds - int
cookieRefresh - bool
usesNonCookieAccess - bool
deviceStorageDisclosureUrl - string

in models of all PRs?

Reply:
Oh, it looks like I didn't push them)